blocks/header/header.css

```diff
@@ -13,44 +19,75 @@ header nav {
     'hamburger brand tools' var(--nav-height)
     'sections sections sections' 1fr / auto 1fr auto;
   align-items: center;
-  gap: 0 2em;
   margin: auto;
-  max-width: 1264px;
+  max-width: 1280px;
```

Comment:
shouldn't this be 1200px, as it's the new breakpoint for desktop?

Reply:
so this is for max-width, for bigger screens we don't want to make header width more than 1280px. I'm not using this as a breakpoint.